aws/cloudformation/s3_buckets.yml

```diff
@@ -20,6 +24,35 @@ Resources:
         BlockPublicPolicy: true
         IgnorePublicAcls: true
         RestrictPublicBuckets: true
+  DroneLibraryBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref LibraryBucket
+      PolicyDocument:
+        Statement:
+          - Sid: Grant Drone permission to objects created by CI tests.
+            Effect: 'Allow'
+            Action:
+              - 's3:*'
+            Resource:
+              - 'arn:aws:s3:::cdo-v3-libraries/libraries_circle/*'
+            Principal:
+              {
+                AWS: [!Sub "arn:aws:iam::${DeveloperAccount}:role/DroneWorker"],
+              }
+          - Sid: Grant Drone permission to list objects for CI tests.
+            Effect: 'Allow'
+            Action:
+              - 's3:ListBucket*'
+            Resource:
+              - 'arn:aws:s3:::cdo-v3-libraries'
+            Condition:
+              StringLike:
+                - 's3:prefix': 'libraries_circle/*'
```

Comment:
```suggestion
                - 's3:Prefix': 'libraries_circle/*'
```

Reply:
i ran the changeset again with this change and got the same failure. i'll look into the syntax, but let me know if you see anything else that needs to change